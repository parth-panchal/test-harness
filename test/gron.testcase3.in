{
  "name": "John Doe",
  "age": 30,
  "city": "New York",
  "skills": ["Python", "JavaScript", "SQL"],
  "contact": {
    "email": "john.doe@example.com",
    "phone": "+1 123-456-7890"
  }
}
