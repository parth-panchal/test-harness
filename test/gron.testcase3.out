o = {};
o.age = 30;
o.city = "New York";
o.contact = {};
o.contact.email = "john.doe@example.com";
o.contact.phone = "+1 123-456-7890";
o.name = "John Doe";
o.skills = [];
o.skills[0] = "Python";
o.skills[1] = "JavaScript";
o.skills[2] = "SQL";
